headers/types/common/common.h

```diff
@@ -104,4 +124,4 @@ ASSERT_SIZE(struct move_data_table, 14534);
 // TODO: Add more data file structures, as convenient or needed, especially if the load address
 // or pointers to the load address are known.
 
-#endif
```

Comment:
Err, let's keep the empty line at the end of the file as is. It makes things look a little nicer in the GitHub viewer.